packages/terra-base/src/Base.scss

```diff
@@ -10,7 +13,10 @@ html {
 }
 
 body {
+  // stylelint-disable order/properties-alphabetical-order
+  background-image: var(--terra-base-background-image, none);
```

Comment:
We need background-image to be applied before background-color so needed to disable this lint rule.